Foundation.md

```diff
@@ -0,0 +1,34 @@
+# Foundation
+
+Foundation is the most advanced responsive front-end framework in the world (According to their website).
```

Comment:
Lol! But seriously, change it something like

> Foundation is one of the most advanced responsive front-end frameworks.

And also link the word _Foundation_ to the homepage of Foundation.